src/containers/legacy-roles/legacy-role.tsx

```diff
@@ -0,0 +1,412 @@
+import { t, Trans } from '@lingui/macro';
+import * as React from 'react';
+import './legacy-roles.scss';
+
+import { Link, RouteComponentProps, withRouter } from 'react-router-dom';
+
+import {
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  LabelGroup,
+  Nav,
+  NavItem,
+  NavList,
+  Panel,
+  TextContent,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import {
+  Breadcrumbs,
+  ClipboardCopy,
+  DateComponent,
+  LoadingPageWithHeader,
+  Logo,
+  Main,
+  Tag,
+} from 'src/components';
+
+import { Paths, formatPath } from 'src/paths';
+import { LegacyRoleAPI } from 'src/api/legacyrole';
+import { AppContext } from 'src/loaders/app-context';
+import { LegacyRoleDetailType } from 'src/api/response-types/legacy-role';
+import { LegacyRoleVersionDetailType } from 'src/api/response-types/legacy-role';
+
+interface RoleMeta {
+  role: LegacyRoleDetailType;
+  github_user: string;
+  name: string;
+  id: number;
+}
+
+interface RoleMetaReadme {
+  readme_html: string;
+}
+
+class LegacyRoleInstall extends React.Component<RoleMeta, RoleMeta> {
+  render() {
+    const installCMD = `ansible-galaxy role install ${this.props.github_user}.${this.props.name}`;
+    return (
+      <>
+        <h1>Installation:</h1>
+        <ClipboardCopy isCode isReadOnly variant={'expansion'}>
+          {installCMD}
+        </ClipboardCopy>
+      </>
+    );
+  }
+}
+
+class LegacyRoleDocs extends React.Component<RoleMeta, RoleMetaReadme> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      readme_html: null,
+    };
+  }
+
+  componentDidMount() {
+    const url = 'roles/' + this.props.role.id + '/content';
+    LegacyRoleAPI.get(url).then((response) => {
+      this.setState(() => ({
+        readme_html: response.data.readme_html,
+      }));
+    });
+  }
+
+  render() {
+    return (
+      <div className='legacy-role-readme-container'>
+        <div
+          className='pf-c-content'
+          dangerouslySetInnerHTML={{ __html: this.state.readme_html }}
+        ></div>
+      </div>
+    );
+  }
+}
+
+interface RoleVersionIProps {
+  role_version: LegacyRoleVersionDetailType;
+}
+
+class LegacyRoleVersion extends React.Component<
+  RoleVersionIProps,
+  RoleVersionIProps
+> {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <DataListItemRow>
+        <DataListCell alignRight>
+          {this.props.role_version.version}
+        </DataListCell>
+
+        <DataListCell alignRight>
+          <Trans>
+            Released <DateComponent date={this.props.role_version.created} />
+          </Trans>
+        </DataListCell>
+
+        {/* Release tarballs hosted on github */}
+        <DataListCell alignRight>
+          <Link to={this.props.role_version.download_url}>
```

Comment:
```suggestion
          <a href={this.props.role_version.download_url} target='_blank' rel='noreferrer'>
```

otherwise this creates links to `http://localhost:8002/ui/legacy/roles/geerlingguy/docker/https://github.com/geerlingguy/ansible-role-docker/archive/6.0.4.tar.gz`